crates/uv-fs/src/lib.rs

```diff
@@ -104,6 +104,26 @@ pub fn remove_symlink(path: impl AsRef<Path>) -> std::io::Result<()> {
     fs_err::remove_file(path.as_ref())
 }
 
+/// Create a symlink at `dst` pointing to `src` or, on Windows, copy `src` to `dst`.
+///
+/// If working with a directory, use [`replace_symlink`] instead which will use a junction on
+/// Windows.
```

Comment:
This doesn't seem to describe what the function below does. (The first sentence does, though.)

Reply:
I believe I've addressed this by tweaking the wording.